tests/appsec/iast/sink/test_iast_ldap_injection.py

```diff
@@ -11,7 +11,8 @@
 
 # Weblog are ok for nodejs/express4 and java/spring-boot
 @coverage.basic
-@released(dotnet="?", golang="?", java="?", nodejs="?", php_appsec="?", python="?", ruby="?")
+@released(dotnet="?", golang="?", nodejs="?", php_appsec="?", python="?", ruby="?")
+@released(java={"spring-boot": "1.7.0", "*": "?"})
```

Comment:
Can we enable it for other variants based on Spring Boot? (can be a separate PR if needed, since this is about sinks)

Reply:
I will create another PR fixing the Xpassed test for different springboot scenarios